main_COPY_AND_RENAME.py

```diff
@@ -7,20 +7,25 @@
 """
 pygame.init()
 
-screen_width, screen_height = 1000, 800
-screen = pygame.display.set_mode((screen_width, screen_height))
-
-clock = pygame.time.Clock()
-FRAME_RATE = 40
+# Global constants
+SCREEN_WIDTH = 1000
+SCREEN_HEIGHT = 800
+FRAME_RATE = 60
 
+# Useful colors 
 BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+
+# Creating the screen and the clock
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+clock = pygame.time.Clock()
 
 while True:
     """
     EVENTS section - how the code reacts when users do things
     """
     for event in pygame.event.get():
-        if event.type == pygame.QUIT:
+        if event.type == pygame.QUIT: # When user clicks the 'x' on the window, close our game
```

Comment:
PEP8 recommends adding a 2nd space between code and comments
```suggestion
        if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
```